integrations/erc20/transfer.go

```diff
@@ -16,6 +16,7 @@ var sig, sigHash []byte
 
 func init() {
 	var err error
+	// Obtained by inputing Transfer(address,address,uint256) into https://emn178.github.io/online-tools/keccak_256.html
 	sig, err = hex.DecodeString("ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")
```

Comment:
Perhaps we should just keccak hash the signature. `init()` is called on program initialization so there isn't a performance concern.

I am also realizing that the double keccak hash isn't clear. We double hash it because that's what's needed when accessing the bloom filter. And it's inefficient to hash the data inside the bloom filter check. For now, maybe we should just do this:

```go
sigHash = isxhash.Keccak(isxhash.Keccak([]byte("Transfer(address,address,uint256)")))
```

Up to you if you want to do this. I can get to it in a later PR. Don't want to punish your good deed ;)

Reply:
Yeah this would definitely be the cleanest, took me a while to figure out what's going on there, thanks for the explanation!

I just made the suggested change also (minor difference is that `sig` is also used in `Insert` so we set sig = hash(event signature) first)